src/components/backdrop/backdrop.js

```diff
@@ -66,6 +71,13 @@ angular
         }
       });
 
+      function resize() {
```

Comment:
1. That seems like a lot of CSS for something so simple.
2. My intention here was just to fix the issue, not refactor the backdrop. If it works like this, it probably has a good reason to. 
3. I can't really see viewport units being used anywhere else in the project.